### PR TITLE
Lesson 6

### PR DESCRIPTION
**Pull request by bahajack**
Create a pull request

### CODE REVIEW

File: src/InputWithLabel.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React, {Fragment} from 'react';
+
+
+const InputWithLabel = (props
+//{
```
Comment (jkylehumphrey) [inline src/InputWithLabel.js]: Remove commented code

File: src/InputWithLabel.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  return(
+
+    <Fragment>
+
+      <label htmlFor="todoTitle">{props.children}</label>
+      &nbsp;
+      <input
+        ref={inputRef}
+        //id={id}
```
Comment (jkylehumphrey) [inline src/InputWithLabel.js]: Remove commented code

File: src/AddTodoForm.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <form onSubmit={handleAddTodo}>
+    <InputWithLabel
+
+      id="search"
+      //value={todoTitle}
+      todoTitle={todoTitle}
+      handleTitleChange={handleTitleChange}
+      //onInputChange={handleTitleChange}
+    >
+    <strong>Search:</strong>
```
Comment (jkylehumphrey) [inline src/AddTodoForm.js]: Is "Search" an accurate label?

**Review by jkylehumphrey — CHANGES_REQUESTED** [review]
Sorry I'm behind on PR's.  Please make the requested changes and merge this PR to `main` before braching for the next lesson

**Comment by bahajack** [comment]
create 2nd pull for lesson6

File: src/AddTodoForm.js (hunk 3 above)
Comment (bahajack) [reply src/AddTodoForm.js]: I think I made all the required changes.

**Pull request merged**